platform/cluster.go

```diff
@@ -151,7 +151,7 @@ func (bc *BaseCluster) RenderUserData(userdata *platformConf.UserData, ignitionV
 		}
 	}
 
-	conf, err := userdata.Render(bc.bf.ctPlatform)
+	conf, err := userdata.Render(bc.bf.ctPlatform, bc.IgnitionVersion() == "v2")
```

Comment:
We should probably keep track of this in the future and make sure that if we start repeating this we provide a cleaner interface for determining if a cluster is expecting Ignition v2. Fine punting that to a later PR though.